frontend/src/main/components/Ride/RideAssignDriverForm.js

```diff
@@ -1,13 +1,60 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { Button, Form } from 'react-bootstrap';
 import { useForm } from 'react-hook-form'
 import { useNavigate } from 'react-router-dom';
-
+import { useBackend } from 'main/utils/useBackend';
 
 
 function RideAssignDriverForm({ initialContents, submitAction, buttonLabel = "Assign Driver" }) {
     const navigate = useNavigate();
     
+    const { data: drivers, _error2, _status2 } =
+        useBackend(
+            // Stryker disable next-line all : don't test internal caching of React Query
+            [`/api/drivers/all`],
+            {  // Stryker disable next-line all : GET is the default, so changing this to "" doesn't introduce a bug
+                method: "GET",
+                url: `/api/drivers/all`
+                // Stryker restore all
+            },
+        );
+
+    const { data: shifts, _error1, _status1 } =
+        useBackend(
+            // Stryker disable next-line all : don't test internal caching of React Query
+            [`/api/shift/all`],
+            {  // Stryker disable next-line all : GET is the default, so changing this to "" doesn't introduce a bug
+                method: "GET",
+                url: `/api/shift/all`
+                // Stryker restore all
+            },
+        );
+
+    const [driverShift, setDriverShift] = useState([]);
+
+    // Stryker disable all
+    useEffect(() => {
+        if (drivers && shifts && drivers.length > 0 && shifts.length > 0) {
+            setDriverShift([]);
+            var driverMap = new Map();
+
+            drivers.forEach(driver => {
+                driverMap.set(driver.id, driver.fullName);
+            });
+
+            shifts.forEach(shift => {
+                setDriverShift(prevState => [...prevState, { 
+                    id: shift.id, 
+                    driverName: driverMap.get(shift.driverID),
+                    day: shift.day,
+                    shiftStart: shift.shiftStart,
+                    shiftEnd: shift.shiftEnd,
+                }]);
+            });
+        }
+    }, [drivers, shifts]);
+    // Stryker enable all
```

Comment:
```suggestion
    // Stryker restore all
```

It turns out `enable all` isn't a thing.